Features no longer needed

https://github.com/melvyn2/wireframe-swraster/blob/8ee097d59433c1ad160ed3b0e12396da92368328/src/main.rs#L1

destructuring_assignment has been stable since 1.59


option_expect_none gives 
```
error[E0635]: unknown feature `option_expect_none`
```

I am currently using rust 1.62

--- a/src/main.rs
+++ b/src/main.rs
@@ -1,5 +1,3 @@
-#![feature(destructuring_assignment)]
-#![feature(option_expect_none)]
 #![feature(test)]
 #![allow(dead_code)]
 

